tests/routes/test_checks.py

```diff
@@ -59,3 +61,49 @@ def test_unavailable(self):
             id='conman.routes.E002',
         )
         self.assertEqual(errors, [error])
+
+
+class SubclassesInAdminTest(SimpleTestCase):
+    """Test checks.subclasses_in_admin."""
+    def test_registered(self):
+        """checks.subclasses_in_admin is a registered check."""
+        registered_checks = registry.get_checks()
+        self.assertIn(checks.subclasses_in_admin, registered_checks)
+
+    def test_in_admin(self):
+        """When a Route subclass is in the admin, show no error."""
+        errors = checks.subclasses_in_admin(app_configs=None)
+        self.assertEqual(errors, [])
+
+    def test_not_in_admin(self):
+        """When a Route subclass isn't in the admin, show an error."""
+        # Store the registered admin class for teardown.
+        admin_class = admin.site._registry[NestedRouteSubclass].__class__
+        try:
+            admin.site.unregister(NestedRouteSubclass)
+            result = checks.subclasses_in_admin(app_configs=None)
+        finally:
+            # Restore the admin to pre-test status.
+            admin.site.register(NestedRouteSubclass, admin_class)
+
+        expected = Error(
+            'Route subclasses missing from admin.',
+            hint="Missing: {<class 'tests.models.NestedRouteSubclass'>}.",
+            id='conman.routes.E003',
+        )
+        self.assertEqual(result, [expected])
+
+    def test_admin_disabled(self):
+        """When the admin isn't active, don't force use of it."""
+        admin_class = admin.site._registry[NestedRouteSubclass].__class__
+
+        path = 'conman.routes.checks.apps.is_installed'
```

Comment:
Perhaps a comment to explain that `apps` is imported in `checks`, so we're actually mocking part of Django, not conman.